From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Apr 2016 16:40:34 +0200
Subject: use_dejavu_as_default_font

Use DejaVu Sans as default font and adjusts sizes accordingly (where
needed).
---
 kdeui/kernel/kglobalsettings.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kdeui/kernel/kglobalsettings.cpp b/kdeui/kernel/kglobalsettings.cpp
index 2e3a7eb..065cef4 100644
--- a/kdeui/kernel/kglobalsettings.cpp
+++ b/kdeui/kernel/kglobalsettings.cpp
@@ -396,7 +396,7 @@ struct KFontData
 
 // NOTE: keep in sync with kdebase/workspace/kcontrol/fonts/fonts.cpp
 static const char GeneralId[] =      "General";
-static const char DefaultFont[] =    "Sans Serif";
+static const char DefaultFont[] =    "DejaVu Sans";
 #ifdef Q_WS_MAC
 static const char DefaultMacFont[] = "Lucida Grande";
 #endif
@@ -415,7 +415,7 @@ static const KFontData DefaultFontData[KGlobalSettingsData::FontTypesCount] =
     { GeneralId, "menuFont",    DefaultFont, 16, -1, QFont::SansSerif },
 #else
     { GeneralId, "font",        DefaultFont, 9, -1, QFont::SansSerif },
-    { GeneralId, "fixed",       "Monospace", 9, -1, QFont::TypeWriter },
+    { GeneralId, "fixed", "DejaVu Sans Mono",  9, -1, QFont::TypeWriter },
     { GeneralId, "toolBarFont", DefaultFont,  8, -1, QFont::SansSerif },
     { GeneralId, "menuFont",    DefaultFont, 9, -1, QFont::SansSerif },
 #endif
